### ISSUE TITLE
Profile not remembered across restart

### ISSUE THREAD

**Issue by NicholasConrad** [digitalcreations/MaxTo#206]
I have two external monitors; one of them is always assigned its previous profile when MaxTo launches on system start, but one of the monitors is always assigned a 'blank' (single region) profile on system start and I have to re-assign its profile every time I restart windows. When I do assign the profile, two of the areas are reversed, and I have to then  re-assign the profile to get the areas in the right order.

Windows 10 Pro v1709
Surface Pro 4

**Comment by vegardlarsen** [comment]
Thank you for writing. This is a known issue due to the way MaxTo currently works with identifying monitors and saving its settings. We have a new way of identifying monitors and saving settings coming up, and this will be fixed.

**Comment by darana** [comment]
That's good to hear. This data point may not be of any help if you've already addressed, but sharing it nontheless. I have four monitors, monitors one and two are in standard landscape mode and never forget their profiles. Monitors 3 and 4 are both in portrait mode, rotated -90 degrees, and both *intermittently* and with no pattern I have been able to discern, forget either their primary, secondary, both, or neither profiles. It only happens across a restart, as noted. Quitting maxto and relaunching it doesn't ever seem to happen. 

Windows 10 x64, latest updates, Nvidia GTX 1070, latest drivers, latest version of MaxTo.

**Comment by vegardlarsen** [comment]
It is probably related to when Windows (or your video driver) attaches and detaches the monitors. If the monitor appears to be disconnected when MaxTo is running today, its settings will be forgotten.

This will not happen with the update I mentioned earlier.

**Comment by killroy42** [comment]
Just to leave a note here. For me, MaxTo always reset to default after nVidia driver updates, and I had to manually reload my settings (twice, since on first load it slightly gets the regions wrong). It also happens when disabling and reenabling the nVidia graphics card in device Manager (which I had to do to fix a LoL issue). Might make reproduction easier. I presume it's the same issue. Can't wait for the update to save me a few minutes every week or so :)
